covid_xprize/validation/scenario_generator.ipynb

```diff
@@ -38,8 +40,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATA_FILE = \"tests/fixtures/OxCGRT_latest.csv\"\n",
-    "latest_df = get_raw_data(DATA_FILE, latest=True)"
+    "LATEST_DATA_URL = 'https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv'\n",
+    "GEO_FILE = \"../../countries_regions.csv\"\n",
+    "latest_df = load_dataset(LATEST_DATA_URL, GEO_FILE)"
```

Comment:
Now using `countries_regions.csv` to get latest data only for these countries and regions.